packages/portal-components/package.json

```diff
@@ -56,7 +56,8 @@
         "url": "https://github.com/fremtind/jokul/issues"
     },
     "dependencies": {
-        "@fremtind/jkl-radio-button-react": "^1.3.5"
+        "classnames": "^2.2.6",
```

Comment:
Why this?

Reply:
It was used by the component without being a dependency, so I added it. Should probably have been it's own commit 😅